consensus/view_change.go

```diff
@@ -183,21 +151,26 @@ func (consensus *Consensus) startViewChange(viewID uint64) {
 			p2p.ConstructMessage(msgToSend),
 		)
 	}
-
-	consensus.consensusTimeout[timeoutViewChange].SetDuration(duration)
-	consensus.consensusTimeout[timeoutViewChange].Start()
 }
 
+// onViewChange is called when the view change message is received.
 func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
+	consensus.getLogger().Debug().Msg("[onViewChange] Received ViewChange Message")
```

Comment:
Let's make all the view change related log Info()? view change doesn't happen all the time, but when it happens, it's critical have the log for troubleshooting.

Reply:
make senses.